<ui version="4.0" >
 <class>MaestroBase</class>
 <widget class="QMainWindow" name="MaestroBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Maestro Client by Infiscape</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="MaestroResource.qrc" >:/Maestro/images/maestro_icon.png</iconset>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QFrame" name="mToolbox" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>1</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth" >
         <number>3</number>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="mStack" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QWidget" name="mOldPage" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>557</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionReload" />
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSave_As" />
    <addaction name="mActionArchiveLogs" />
    <addaction name="separator" />
    <addaction name="action_Exit" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="mToolbar" >
   <property name="windowTitle" >
    <string>Maestro Toolbar</string>
   </property>
   <property name="movable" >
    <bool>true</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionReload" />
   <addaction name="actionNew" />
   <addaction name="actionOpen" />
   <addaction name="actionSave" />
  </widget>
  <widget class="QDockWidget" name="mLogWindow" >
   <property name="allowedAreas" >
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Log Window</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="mDockWidgetContents" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>9</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
    </layout>
   </widget>
  </widget>
  <action name="actionReload" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>&amp;Reload</string>
   </property>
  </action>
  <action name="actionNew" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As" >
   <property name="text" >
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="action_Exit" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_About" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="mActionArchiveLogs" >
   <property name="text" >
    <string>Archive Logs...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MaestroResource.qrc" />
 </resources>
 <connections/>
</ui>
